config/config.php

```diff
@@ -254,82 +125,19 @@
                 ),
             ),
         ),
-        'message' => array(
-            'icon' => 'static/apps/noviusos_form/img/fields/message.png',
-            'title' => __('Message'),
-            'definition' => array(
-                'layout' => 'message=4',
-                'fields' => array(
-                    'message' => array(
-                        'driver' => \Nos\Form\Driver_Field_Message::class,
-                        'default_values' => array(
-                            'field_label' => __('Message:'),
-                            'field_message' => 'Your message',
-                        ),
-                    ),
-                ),
-            ),
-        ),
-        'separator' => array(
-            'icon' => 'static/apps/noviusos_form/img/fields/separator.png',
-            'title' => __('Separator'),
-            'definition' => array(
-                'layout' => 'separator=4',
-                'fields' => array(
-                    'separator' => array(
-                        'driver' => \Nos\Form\Driver_Field_Separator::class,
-                        'default_values' => array(
-                            'field_label' => __('Separator'),
-                        ),
-                    ),
-                ),
-            ),
-        ),
-        'recipients' => array(
-            'icon' => 'static/apps/noviusos_form/img/fields/dropdown.png',
-            'title' => __('Email Recipient List'),
-            'definition' => array(
-                'layout' => 'select=4',
-                'fields' => array(
-                    'select' => array(
-                        'driver' => \Nos\Form\Driver_Field_Recipient_Select::class,
-                        'default_values' => array(
-                            'field_technical_id' => 'recipient-list',
-                            'field_details' => __('A notification of the form answer will be sent to the selected email.'), //Separate name and email addresses with a "=" sign. (eg. Name=mail@domain.com)
-                            'field_choices' => __("First option=mail@domain.com\nSecond option=othermail@domain.com"),
-                        ),
-                    ),
-                ),
-            ),
-        ),
-        'hidden' => array(
-            'icon' => 'static/apps/noviusos_form/img/fields/hidden.png',
-            'title' => __('Hidden'),
-            'expert' => true,
```

Comment:
I think this "hidden" field is no longer defined as "expert only" since you remove this. The same for variable field.
We have to add an "expert" key in these fields configuration.